src/helpers/color.js

```diff
@@ -46,7 +46,9 @@ export default {
   },
 
   isValidHex(hex) {
-    return tinycolor(hex).isValid()
+    // disable hex4 and hex8
+    const lh = (String(hex).charAt(0) === '#') ? 1 : 0
```

Comment:
Minor point, but I'm curious: why isn't this `hex[0] == '#'`? Since length is checked, `hex` is expected to be a string.

Another alternative:
```js
const len = hex.startsWith('#') ? hex.length - 1 : hex.length
return (len == 3 || len == 6) && tinycolor(hex).isValid()
```

By checking that length is 3 or 6, instead of NOT 4 or >6 (per the spec of this function), it won't matter if tinycolor later allows other lengths.